### ISSUE TITLE
Prevent .dovecot.sieve folder from showing up in the IMAP folder list

### ISSUE THREAD

**Issue by theomega** [tomav/docker-mailserver#508]
I have sieve enabled, and I'm using a Roundcube plugin to configure it. What happens is that for the user I'm configuring, the following symlink is created in the `/var/mail/[domain]/[user]`folder:
`.dovecot.sieve -> sieve/roundcube.sieve`

This leads into my IMAP clients showing a `dovecot` folder with a `sieve` subfolder because the interpret everything in the email folder starting with a dot as an folder. 

Did someone manage to prevent this?

This mailinglist posting describes that you should not put the sieve files into the same folder as the emails. They suggest to have a common root and then two subfolders: one for the emails and the other one for sieve. They also strongly suggest to have real home directories:
http://www.dovecot.org/pipermail/dovecot/2011-June/059875.html

**Comment by flole** [comment]
I added the following to config/dovecot.cf
```
plugin {
  sieve = /var/mail/sieve/%d/%n/.dovecot.sieve
  sieve_dir = /var/mail/sieve/%d/%n/sieve
}
```

This keeps the sieve files out of the maildir

**Comment by theomega** [comment]
Perfect, works!

Maybe we should make this the default? The original settings lead to the effect I was describing and also make add some error messages about permissions when dovecot tries to access the sieve files and thinks they are imap folders.

**Comment by flole** [comment]
I had the same problems with Apple Mail. It showed the dovecot folder and when I tried to open it, an error was shown.

Make this the default would be great for all who has the same problem.

**Comment by tomav** [comment]
Yep, thanks for spotting that. You can submit a PR and I'll merge it.

**Comment by theomega** [comment]
The problem here is the migration path: People already have their sieve scripts in the "old" location. So one would need to move the files to the right locations.

**Comment by johansmitsnl** [comment]
#651 also addresses this.
There is a PR #654 in the making but not working fully yet.

**Comment by erik-wramner** [comment]
@theomega @johansmitsnl what happened with this? The other issue is also still open and the PR for the other issue is closed with the final comment "I'll patch up your pr". Does that mean that both issues can be closed or that the problem persists?

**Comment by johansmitsnl** [comment]
@erik-wramner it was still an issue, not sure why I closed the PR back then.

**Comment by erik-wramner** [comment]
Nothing has happened with this for a long time, but reading it again isn't it just documentation? There seems to be a solution with config/dovecot.cf that works and that is simple to implement.

**Comment by aendeavor** [comment]
I closed #651 as a duplicate of this issue. The matter does not seem to be resolved however. We would need a PR for this.

**Comment by aendeavor** [comment]
This issue was closed due to one or more of the following reasons:

1. Age
2. Contributor inactivity
3. The issue seems to be resolved

If you think this happened by accident, or feel like this issue was not actually resolved, please feel free to re-open it. If there is an issue you could resolve in the meantime, please open a PR based on the current master branch so we can review it.
